normalize.css

```diff
@@ -0,0 +1,461 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
there are at least 2 better ways to get `normalize.css` into your project. Either use a CDN or `npm`